client/signup/steps/domains/index.jsx

```diff
@@ -173,6 +176,11 @@ export class RenderDomainsStep extends Component {
 			);
 		}
 
+		// We add a plan to cart on Multi Domains to show the proper discount on the mini-cart
+		if ( shouldUseMultipleDomainsInCart( this.props.flowName ) ) {
```

Comment:
My only concern here is when we decide to release the multi-domain feature, we'd need to ensure this code is only called in the multi-domain context.

In other words, we'll probably want to only remove the [isEnabled check from this method](https://github.com/Automattic/wp-calypso/blob/trunk/client/signup/steps/domains/utils.js#L40), instead of removing its checks all over the place. Does that make sense?